client/app/components/packages/filed-eas/edit.js

```diff
@@ -5,6 +5,18 @@ import SubmittablePackageFormValidations from '../../../validations/submittable-
 import SubmittableCeqrInvoiceQuestionnaireFormValidations from '../../../validations/submittable-ceqr-invoice-questionnaire-form';
 
 export default class PackagesFiledEasEditComponent extends Component {
+  @service
+  store;
+
+  constructor(...args) {
+    super(...args);
+
+    if (this.args.package.ceqrInvoiceQuestionnaires.length < 1) {
+      const newCeqrInvoiceQuestionnaire = this.store.createRecord('ceqr-invoice-questionnaire');
+      this.args.package.ceqrInvoiceQuestionnaires.pushObject(newCeqrInvoiceQuestionnaire);
```

Comment:
Would there ever be a case that the package hasn't loaded yet by the time this component renders? 
Probably not since Ember waits for route models to finish loading. But this is something to watch out for..

Reply:
Interesting point, I'm not totally sure -- I can try to look into it and if it becomes a problem we can update this? What would be your suggestion for avoiding that?

Reply:
The package could still be a promise, but in general we don't deal with asynchronous model relationships and prefer to let the route resolve those dependencies first.